package.json

```diff
@@ -60,7 +60,7 @@
     "hash-sum": "^1.0.2",
     "lru-cache": "^4.1.2",
     "merge-source-map": "^1.1.0",
-    "postcss": "^7.0.36",
+    "postcss": "^8.0.0",
```

Comment:
Why version `8.0.0`? I would use version `^8.2.13` or higher.

Reply:
I can change it if you want, but anything with `^` at the beginning will evaluate to the same version (newest one starting with `8.`), so there is no difference

Reply:
Being more specific can be helpful to signal where the actual requirement is, but you're not wrong.

Reply:
Hard to tell whether I should change it or not - would be great to have repo maintainers express their opinion on this one (and the whole pr)

Reply:
Since the vulnerability was patched in 8.1.3, IMO we should at least require that version.

Reply:
Sure, bumped to the newest version just now